packages/web-frontend/src/screens/LandingPage/LandingPage.js

```diff
@@ -23,7 +23,7 @@ const Wrapper = styled.div`
   position: relative;
   background-size: cover;
   background-position: center;
-  background-color: #000000;
+  background-color: #262834;
```

Comment:
Maybe just check this with Felicity - one of the tickets said somewhere the default background was black?

Reply:
Oh yes you are right. I found black quite jarring when I was testing so went for this which is used in the loader. I've added a note for Flic in the ticket to check.